modules/openxBidAdapter.js

```diff
@@ -248,6 +248,31 @@ function buildCommonQueryParamsFromBids(bids, bidderRequest) {
     nocache: new Date().getTime()
   };
 
+  const firstPartyData = config.getConfig('ortb2.user.data')
+  if (Array.isArray(firstPartyData) && firstPartyData.length > 0) {
+    // extract and merge valid segments by provider/taxonomy
+    const fpd = firstPartyData
+      .filter(
+        data => (Array.isArray(data.segment) &&
+                data.segment.length > 0 &&
+                data.name !== undefined &&
+                data.name.length > 0)
+      )
+      .reduce((acc, data) => {
+        const name = typeof data.ext === 'object' && data.ext.segtax ? `${data.name}/${data.ext.segtax}` : data.name;
+        acc[name] = (acc[name] || []).concat(data.segment.map(seg => seg.id));
+        return acc;
+      }, {})
+    const sm = Object.entries(fpd)
```

Comment:
The tests are failing in IE11 because Object.entries is not supported. Can you please use a polyfill. One way you can solve it by using a polyfill from `core-js-pure/features/object/entries`. If you use this, remember to add it to `allowedModules`

Another way could be you use a polyfill in your adapter instead of importing the one above.

Reply:
Thank you, I will rewrite it using `Object.keys()` if IE9 support is not required.